net/tests/Sails.TestUtils/Containers/GearNodeContainer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using EnsureThat;
+
+namespace Sails.TestUtils.Containers;
+
+public sealed class GearNodeContainer : IAsyncDisposable
+{
+    public GearNodeContainer(string gearNodeVersion, bool reuse)
```

Comment:
wouldn't it be better to expose `GearNodeBuilder` ? like other testcontainers

Reply:
added TODO for consideration